zap/src/main/java/org/zaproxy/zap/extension/sessions/SessionManagementAPI.java

```diff
@@ -65,8 +65,6 @@ public class SessionManagementAPI extends ApiImplementor {
     @SuppressWarnings("unused")
```

Comment:
This should be removed, it's being used now.

Reply:
Done